edb/pgsql/compiler/clauses.py

```diff
@@ -256,8 +319,12 @@ def compile_orderby_clause(
 
             # In OPDER BY we compile ir.Set as a subquery:
```

Comment:
Typo fix opportunity:

```suggestion
            # In ORDER BY we compile ir.Set as a subquery:
```